Userland/Games/Minesweeper/Field.cpp

```diff
@@ -278,14 +324,14 @@ void Field::reset()
             });
             square.number = number;
             if (square.has_mine)
-                continue;
-            if (square.number)
+                square.label->set_icon(m_mine_bitmap);
+            else if (square.number) {
                 square.label->set_icon(m_number_bitmap[square.number - 1]);
+            }
         }
     }
 
     m_unswept_empties = rows() * columns() - m_mine_count;
-    set_updates_enabled(true);
```

Comment:
...same for this line.

Reply:
This one's just due to how diffing worked, there's another line in the diff below adding it back :D

Reply:
Ahhh, you're right, I missed that. git is weird sometimes. :sweat_smile: